README.md

```diff
@@ -29,9 +29,11 @@ An example [Python script](https://github.com/Parquery/mailgun-relayery/example.
 Usage
 =====
 
-Preliminary steps
------------------
-*  Run the [release script](https://github.com/Parquery/mailgun-relayery/release.py) to compile and release the two Go 
+Compilation
+-----------
+TODO: add links to pre-compiled binaries.
+
```

Comment:
(Related to the comment from a previous review regarding the extra tool to initialize the database)
Please add a TODO for the initialization tool. Indeed, it would make most sense to me to make it a proper Go-compiled binary and include it in the release. I'd prefix its name appropriately (mailgun-relayery-init? mailgun-relayer-create?).

Reply:
makes sense, will make a separate PR.